code/modules/client/preferences_gear.dm

```diff
@@ -830,7 +845,7 @@ var/global/list/gear_datums = list()
 	path = /obj/item/clothing/suit/unathi/robe
 	cost = 1
 	slot = slot_wear_suit
-//	whitelisted = "Unathi" // You don't have a monopoly on a robe!
+	whitelisted = "Unathi"
```

Comment:
I have seen non unathi players using and enjoying these robes. i dont see any reason it should be unathi restricted. there are plenty of human cultures that like robes